Doc/library/pyexpat.rst

```diff
@@ -196,6 +196,33 @@ XMLParser Objects
    :exc:`ExpatError` to be raised with the :attr:`code` attribute set to
    ``errors.codes[errors.XML_ERROR_CANT_CHANGE_FEATURE_ONCE_PARSING]``.
 
+.. warning::
```

Comment:
It should be nested in the SetReparseDeferralEnabled block. Note that when you open a direct link to SetReparseDeferralEnabled, this warning is outside of the visible area.

Reply:
@serhiy-storchaka good point, I wasn't sure if that kind of nesting worked — nesting is not rst's strong side — but I can can flip it, one second…

Reply:
Done